notebooks/samples/pytorch/iris_classification/serving_pytorch_models_in_ai_platform.ipynb

```diff
@@ -0,0 +1,1195 @@
+{
+ "cells": [
+  {
+   "cell_type": "code",
+   "execution_count": null,
+   "metadata": {},
+   "outputs": [],
+   "source": [
+    "# Copyright 2020 Google LLC\n",
+    "#\n",
+    "# Licensed under the Apache License, Version 2.0 (the \"License\");\n",
+    "# you may not use this file except in compliance with the License.\n",
+    "# You may obtain a copy of the License at\n",
+    "#\n",
+    "#     https://www.apache.org/licenses/LICENSE-2.0\n",
+    "#\n",
+    "# Unless required by applicable law or agreed to in writing, software\n",
+    "# distributed under the License is distributed on an \"AS IS\" BASIS,\n",
+    "# WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.\n",
+    "# See the License for the specific language governing permissions and\n",
+    "# limitations under the License."
+   ]
+  },
+  {
+   "cell_type": "markdown",
+   "metadata": {},
+   "source": [
+    "<table align=\"left\">\n",
+    "  <td>\n",
+    "    <a href=\"https://colab.research.google.com/github/GoogleCloudPlatform/ai-platform-samples/blob/master/notebooks/samples/pytorch/iris_classification/serving_pytorch_models_in_ai_platform.ipynb\"\">\n",
+    "      <img src=\"https://cloud.google.com/ml-engine/images/colab-logo-32px.png\" alt=\"Colab logo\"> Run in Colab\n",
+    "    </a>\n",
+    "  </td>\n",
+    "  <td>\n",
+    "    <a href=\"https://github.com/GoogleCloudPlatform/ai-platform-samples/blob/master/notebooks/samples/pytorch/iris_classification/serving_pytorch_models_in_ai_platform.ipynb\">\n",
+    "      <img src=\"https://cloud.google.com/ml-engine/images/github-logo-32px.png\" alt=\"GitHub logo\">\n",
+    "      View on GitHub\n",
+    "    </a>\n",
+    "  </td>\n",
+    "</table>"
+   ]
+  },
+  {
+   "cell_type": "markdown",
+   "metadata": {
+    "colab_type": "text",
+    "id": "Yy8l-95JkRx0"
+   },
+   "source": [
+    "# Overview\n",
+    "\n",
+    "AI Platform Online Prediction now supports custom python code in to apply custom prediction routines, including custom (stateful) pre/post processing, and/or models not created by the standard supported frameworks (TensorFlow, Keras, Scikit-learn, XGBoost).\n",
+    "\n",
+    "### Dataset\n",
+    "\n",
+    "We use the [Iris dataset](https://archive.ics.uci.edu/ml/datasets/Iris)\n",
+    "\n",
+    "### Objective\n",
+    "\n",
+    "In this notebook, we show how to deploy a model created by [PyTorch](https://pytorch.org/) using AI Platform  Custom Prediction Code using Iris dataset for a multi-class classification problem.\n",
+    "\n",
+    "### Costs \n",
+    "\n",
+    "This tutorial uses billable components of Google Cloud Platform (GCP):\n",
+    "\n",
+    "* Cloud AI Platform\n",
+    "* Cloud Storage\n",
+    "\n",
+    "Learn about [Cloud AI Platform\n",
+    "pricing](https://cloud.google.com/ml-engine/docs/pricing) and [Cloud Storage\n",
```

Comment:
Can you update this link to `https://cloud.google.com/ai-platform/prediction/pricing`? And the link text to `[AI Platform Prediction] pricing?

(Currently link redirects to Training pricing)

Reply:
ack